modules/persistent-storage-local-install.adoc

```diff
@@ -0,0 +1,6 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-storage-install_{context}"]
+= Installing the Local Storage Operator
```

Comment:
Did you try adding `redhat-operator-art` source and see how it works?  I think We should write this document with the assumption that redhat-operator-art source is already added and then document the installation. See - https://bugzilla.redhat.com/show_bug.cgi?id=1733116 for more information.